code/game/machinery/vending.dm

```diff
@@ -143,7 +143,7 @@
 	product_records = null
 	return ..()
 
-/obj/machinery/vending/legacy_ex_act(severity)
+/obj/machinery/vending/ex_act(severity)
```

Comment:
unless you changed this under silicons instructions please revert to legacy_ex_act